src/content/en/updates/2017/09/abortable-fetch.md

```diff
@@ -0,0 +1,305 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Aborting fetches using a new web primitive – The abort controller.
+
+{# wf_updated_on: 2017-09-28 #}
+{# wf_published_on: 2017-09-28 #}
+{# wf_tags: fetch #}
+{# wf_featured_image: /web/updates/images/2017/09/abortable-fetch.png #}
+{# wf_featured_snippet: Aborting fetches using a new web primitive – The abort controller #}
+{# wf_blink_components: Blink>Network>FetchAPI #}
+
+# Abortable fetch {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+The [original GitHub issue for "Aborting a fetch"](https://github.com/whatwg/fetch/issues/27) was
+opened in 2015. Now, if I take 2015 away from 2017 (the current year), I get 2. This demonstrates a
+bug in maths, because 2015 was in fact "forever" ago.
+
+2015 was when we first started exploring aborting ongoing fetches, and after 780 GitHub comments, a
+couple of false starts, and 5 pull requests, we finally have abortable fetch landing in browsers,
+with the first being Firefox 57.
+
+I'll dive into the history later, but first, the API:
+
+## The controller + signal manoeuvre
+
+Meet the `AbortController` and `AbortSignal`:
+
+    const controller = new AbortController();
+    const signal = controller.signal;
+
+The controller only has one method:
+
+    controller.abort();
+
+When you do this, it notifies the signal:
+
+    signal.addEventListener('abort', () => {
+      // Logs true:
+      console.log(signal.aborted);
+    });
+
+This API is provided by [the DOM
+standard](https://dom.spec.whatwg.org/#aborting-ongoing-activities), and that's the entire API. It's
+deliberately generic so it can be used by other web standards and JavaScript libraries.
```

Comment:
FWIW I have asked the MDN folks to prioritize fixing the documentation.  MDN still refers to FetchSignal, etc.

Reply:
Just saw that, thank you!